src/main/java/frc/robot/subsystems/ShooterSystem.java

```diff
@@ -32,11 +43,6 @@ public ShooterSystem() {
         motorLB.restoreFactoryDefaults();
         motorRB.restoreFactoryDefaults();
 
-        motorRT.setIdleMode(CANSparkBase.IdleMode.kBrake);
```

Comment:
don't add changes that are not relevant to the branch. You see a problem: report and fix in a separate branch